tests/chaos/utils.py

```
@@ -139,7 +139,7 @@ def _monitor_nginx_server(
         timeout_watch = TimeoutWatch(timeout=_sampling_duration)
         while timeout_watch.remaining_time() > 0:
             if not is_http_ok(utility_pods=_utility_pods, node=_control_plane_host_node, url=_url):
-                raise Exception("Wrong status code from server.")
+                raise RuntimeError("Wrong status code from server.")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)